controllers/front/OrderController.php

```diff
@@ -237,6 +237,25 @@ public function displayAjaxselectDeliveryOption()
         ]));
     }
 
+    public function displayAjaxCheckCartStillOrderable(): void
+    {
+        $responseData = [
+            'errors' => false,
+            'cartUrl' => '',
+        ];
+
+        // TODO - move to separate single function in Cart and also check if discounts and rules are valid
```

Comment:
We can't keep a PR with a TODO like that, I'm sure in 10 years, there will always be this todo ^^

Reply:
@PierreRambaud So what should I do if I don't want to dive into it right now, but don't want it to be forgotten?

Reply:
I made an issue for this: https://github.com/PrestaShop/PrestaShop/issues/27023